website/content/gateway/api_reference/resources/debits/recipient_information.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "recipient_information"
+date: 2023-07-20T13:01:49+01:00
+anchor: "recipient_information"
+weight: 178
+---
+##### Recipient information
+
+{{% description_list %}}
+
+{{% description_term %}}recipient[name] {{% regex %}}[\x20-\x7E]{2,30} [ASCII printable characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}The recipient's legal name.
+
+Example: "Doe Jane A." (last name, first name, optional middle initial).
+
+{{% regex_optional %}}It must not contain special characters (?, @, #, $, &, \*, etc.), be all numeric, be fictious, be a nickname or be incomplete (only first or last name).{{% /regex_optional %}}
+{{% regex_optional %}}Required for intra-EEA and international debits. Also required if any of the address-related parameters are supplied.{{% /regex_optional %}}
```

Comment:
```suggestion
{{% regex_optional %}}Required for intra-EEA and international debits.{{% /regex_optional %}}
```
No address-related parameters for `recipient`.

Reply:
I will argue that the recipient name is required for all transactions – no matter whether they are domestic, intra EEA or international. :)

Reply:
Really? Wasn't domestic except?

Reply:
From October 2022 it is required according to: 
<img width="541" alt="image" src="https://github.com/clearhaus/gateway-api-docs/assets/47207667/47658865-fd36-4a13-963e-baecc1f59585">

Reply:
Sorry. That was a copy-waste mistake.